controllers/oauth/callback.go

```diff
@@ -80,7 +80,7 @@ func CallbackHandler(conf gooauth.Config) func(c *gin.Context) {
 			return
 		}
 
-		messages.AddInfo("infos", fmt.Sprintf("%s", token))
+		messages.AddInfo("infos", fmt.Sprintf("%s", token.Raw))
```

Comment:
@akuma06 this line, alone other lines using wrong values during string formatting, started creating errors a few hours ago on every PR
token itself is not a string but features a .Raw that is a string (and also a .Signature, along a few other variables that are not string)
Is that the correct thing to do, showing the raw token? I know nothing about oauth and how it works so i'm unsure if that's the right thing to do